openstack/orchestration/v1/stacks/requests.go

```diff
@@ -133,20 +139,6 @@ type AdoptOpts struct {
 	Name string
 	// (REQUIRED) The timeout for stack creation in minutes.
 	Timeout int
-	// (OPTIONAL; REQUIRED IF Template IS EMPTY) The URL of the template to instantiate.
```

Comment:
Actually, it looks like the api still accepts `template` and `templateURL`, but gives preference to `adopt-stack-data`. I'll confirm this tomorrow. If that's the case, we can leave this interface as-is, and simply not provide options for `template` or `templateURL` in the commands in rack.